modules/ding_mobilesearch/ding_mobilesearch.install

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the MobileSearch module.
+ */
+
+/**
+ * Set default configuration upon installation or module update.
+ */
+function ding_mobilesearch_default_config() {
+  $types_to_enable = array('ding_event', 'ding_library', 'ding_news');
```

Comment:
This default config will make these modules requires (hence dependencies to this module).